test/parallel/test-http-keep-alive-drop-requests.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const common = require('../common');
+const http = require('http');
+const net = require('net');
+const assert = require('assert');
+
+function request(socket) {
+  socket.write('GET / HTTP/1.1\r\n');
+  socket.write('Connection: keep-alive\r\n');
+  socket.write('\r\n\r\n');
+}
+
+const server = http.createServer(common.mustCall((req, res) => {
```

Comment:
I think it makes sense to add a test also for an HTTPS server.

Reply:
> I think it makes sense to add a test also for an HTTPS server.

Thanks ! Done !